scripts/pre_commit_linter.py

```diff
@@ -100,6 +100,18 @@
 }
 
 BAD_PATTERNS_JS_REGEXP = [
+    {
+        'regexp': r'\b(browser.explore)\(',
+        'message': "Please remove browser.explore() calls from e2e tests.",
```

Comment:
Please rephrase the message to: `In tests, please do not use browser.explore().`

Reply:
Done!